userprofile/models.py

```diff
@@ -84,6 +92,7 @@ def get_dutytime(self):
                         duty = [DutyTime.objects.create(day=day, start_time=start_time, end_time=end_time)]
                         self.duty = duty
                     break
+                break
```

Comment:
Why use for-loop when you break the first iteration?

Reply:
Because that's the best way I could think of to parse that JSON object. Feel free to find a better way.

Reply:
`time=result[0][0]`

Reply:
Why chat in english when we're all norwegians? ;)

Reply:
Because